client/nui/app.js

```diff
@@ -42,6 +41,15 @@ function close() {
 	}
 }
 
+const copyToClipboard = (str) => {
+  const el = document.createElement("textarea");
+  el.value = str.replace(/[" ]/g, "");
+  document.body.appendChild(el);
+  el.select();
+  document.execCommand("copy");
+  document.body.removeChild(el);
+};
+
```

Comment:
Not a very good implementation, including that `document.execCommand()` has been deprecated. Have a look at https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard instead.